### PR TITLE
Fix: unhandled errors

### PR DESCRIPTION
**Pull request by EnameEtavir**
As these errors where not explicitly ignored with _, I made changes to make sure they bubble up.

### CODE REVIEW

File: pkg/manager/task_stash_box_tag.go [hunk 1]
```diff
@@ -162,6 +162,9 @@ func (t *StashBoxPerformerTagTask) stashBoxPerformerTag() {
 						return err
 					}
 					err = r.Performer().UpdateImage(t.performer.ID, image)
+					if err != nil {
+						return err
+					}
```
Comment (WithoutPants) [inline pkg/manager/task_stash_box_tag.go:167]: This is redundant - error is returned below.

File: pkg/api/resolver_query_find_scene.go [hunk 2]
```diff
@@ -19,6 +19,9 @@ func (r *queryResolver) FindScene(ctx context.Context, id *string, checksum *str
 				return err
 			}
 			scene, err = qb.Find(idInt)
+			if err != nil {
+				return err
+			}
```
Comment (WithoutPants) [inline pkg/api/resolver_query_find_scene.go:24]: Redundant - error is returned below.

File: pkg/api/resolver_query_find_image.go [hunk 3]
```diff
@@ -21,6 +21,9 @@ func (r *queryResolver) FindImage(ctx context.Context, id *string, checksum *str
 			}
 
 			image, err = qb.Find(idInt)
+			if err != nil {
+				return err
+			}
```
Comment (WithoutPants) [inline pkg/api/resolver_query_find_image.go:26]: Redundant: returned below.

**Review by WithoutPants — CHANGES_REQUESTED** [review]

**Comment by EnameEtavir** [comment]
I agree, they are redundant, but from a maintainability view they could help you when you refactor the code or add additional steps where you might not see that those errors were not handled in the scope where the originated.

If you still want it removed, I'll do so, just wanted to bring the point up as food for thought.

**Comment by WithoutPants** [comment]
Decent point. I'll merge as is.

**Pull request merged**